app/templates/introduction.html

```diff
@@ -44,7 +44,7 @@ <h2 class="saturn col-10@l">{{legal_text}}</h2>
               <dd class="dl__data mars">{{ meta.respondent.address.trading_as }}</dd>
             {% endif %}
             <dt id="details-changed-title" class="dl__title mercury">Change in details</dt>
-            <dd class="dl__data mars">Call <a href="tel:0300 1234 931" aria-describedby="details-changed-title">0300 1234 931</a> or email <a href="mailto:surveys@ons.gov.uk?subject=Change%20of%20details%20reference%20{{ meta.respondent.respondent_id }}" aria-describedby="details-changed-title">surveys@ons.gov.uk</a> if there have been any changes to your business name, address or structure</dd>
```

Comment:
we should check what impact the `aria-describerby` has with this change as it used to be specific to the page but is now always the same in the `telephone_contact.html` - may need to pass the value into the template.

Reply:
Now passes the values used from introduction and is configured to pass aria-describedby as a parameter